hubconf.py

```diff
@@ -0,0 +1,57 @@
+import torch
+from models.backbone import Backbone, Joiner
+from models.deformable_detr import DeformableDETR, PostProcess
+from models.position_encoding import PositionEmbeddingSine
+from models.deformable_transformer import DeformableTransformer
+
+dependencies = ["torch", "torchvision"]
+
+
+def _make_deformable_detr(backbone_name: str, dilation=False, num_classes=91):
+    hidden_dim = 256
+    backbone = Backbone(backbone_name, train_backbone=True, return_interm_layers=True, dilation=dilation)
+    pos_enc = PositionEmbeddingSine(hidden_dim // 2, normalize=True)
+    backbone_with_pos_enc = Joiner(backbone, pos_enc)
+    backbone_with_pos_enc.num_channels = backbone.num_channels
+    transformer = DeformableTransformer(d_model=hidden_dim, return_intermediate_dec=True)
+    deformable_detr = DeformableDETR(backbone_with_pos_enc, transformer, num_classes=num_classes, num_queries=300,
+                                     num_feature_levels=4)
+    return deformable_detr
+
+
+def deformable_detr_resnet101(pretrained=False, return_postprocessor=False, checkpoints_path=""):
+    """
+    Deformable DETR R50 with 6 encoder and 6 decoder layers.
+    """
+    num_classes = 91
+    model = _make_deformable_detr("resnet50", dilation=False, num_classes=num_classes)
```

Comment:
Let's make it `resent101` by default

Reply:
I see its a typo in the function name as you are defining `resnet101` model below as well. What we can do is remove the `resent50` model function as we are not using it the paper.